python/matplotlib.ipynb

```diff
@@ -46,63 +46,84 @@
    },
    "outputs": [],
    "source": [
-    "import numpy as np"
+    "import numpy as np\n",
+    "x = np.linspace(0, 1) # gibt 50 Zahlen in gleichmäßigem Abstand von 0–1"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Zu erst ein einfaches Beispiel mit $f(x)=x^2$"
+    "Zu erst ein einfaches Beispiel mit $f(x)=x^2$\n",
+    "\n",
+    "Um den Text-Output zu unterdrücken in diesem Notebook, schreiben wir ein `;` hinter die letzte Zeile.\n",
+    "\n",
+    "Im folgenden verwenden wir die objekt-orientierte Schreibweise von `matplotlib`, die mehr Möglichkeiten und Freiheiten bietet.\n",
+    "\n",
+    "Diese ruft ihr mit"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "tags": []
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "x = np.linspace(0, 1) # gibt 50 Zahlen in gleichmäßigem Abstand von 0–1\n",
-    "plt.plot(x, x**2);\n",
-    "# Falls nicht interaktiv: \n",
-    "# plt.show()"
+    "fig, ax = plt.subplots()\n",
+    "\n",
+    "ax.plot(x, x**2);"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Um den Text-Output zu unterdrücken in diesem Notebook, schreiben wir ein `;` hinter die letzte Zeile."
+    "auf.\n",
+    "\n",
+    "Ihr habt dann ein Objekt figure `fig` und ein Objekt axes `ax`, mit denen ihr interagieren und die Einstellungen im plot vornehmen könnt.\n",
```

Comment:
Figure und Axes sollten hier einmal erklärt werden

* Figure ist die gesamte Abbildung
* Axes ist ein Koordinatensystem in dieser Abbildung, in die man dann Dinge plotten kann